src/Ganeti/HTools/Node.hs

```diff
@@ -1194,9 +1263,10 @@ showField t field =
     "nmem" -> printf "%5d" $ nMem t
     "xmem" -> printf "%5d" $ xMem t
     "fmem" -> printf "%5d" $ fMem t
+    "umem" -> printf "%5d" $ unallocatedMem t
     "imem" -> printf "%5d" $ iMem t
     "rmem" -> printf "%5d" $ rMem t
-    "amem" -> printf "%5d" $ fMem t - rMem t
+    "amem" -> printf "%5d" $ availMem t
```

Comment:
The definition in the doc says available memory is fmem - rmem, but the availMem was bounded by 0. Since this is just in a log and the value bounded by 0 is used in score calculations, I tried to unify this. I've also updated the docs (see doc/design-resource-model.rst)